cdc/model/sink.go

```diff
@@ -87,6 +88,7 @@ type RowChangedEvent struct {
 type Column struct {
 	Type        byte        `json:"t"`
 	WhereHandle *bool       `json:"h,omitempty"`
+	Charset     string      `json:"c"`
```

Comment:
Probably should use int32 or int64 for the sake of extensibility. We may need to store other flags in the future.

Reply:
use `0x0` for binary charset,`0x01` for others?